data/templates/article.jats_publishing

```diff
@@ -109,6 +109,22 @@ $elseif(author.name)$
 $else$
 <string-name>$author$</string-name>
 $endif$
+$if(author.roles)$
```

Comment:
I don't think you need this outer `if`.  If `author.roles` is not defined, `$for(author.roles)$...$endfor$` just won't do anything (iirc).

Reply:
Thanks, I addressed this!